src/services/settings-service/settings-service.ts

```diff
@@ -36,15 +36,15 @@ export class SettingsService {
         return this.theme.asObservable();
     }
 
-    saveNightModeSettings(theme: ThemeType): Promise<void> {
+    saveNightModeSettings(theme: ThemeType): Promise<any> {
         return this.storage.set("night_mode", JSON.parse(JSON.stringify(theme)));
```

Comment:
Why the stringify and parse?

Reply:
Good question, values are saved as string not JSON. Removed next request